src/super_gradients/training/metrics/segmentation_metrics.py

```diff
@@ -111,3 +111,16 @@ def update(self, preds, target: torch.Tensor):
             preds = preds[0]
         _, preds = torch.max(preds, 1)
         super().update(preds=preds, target=target)
+
+
+class BinaryIOU(torchmetrics.IoU):
```

Comment:
@shaydeci  - Did we make a decision on what measures we are going to take to handle the SegmentationMetrics issue we've had with mIOU (caused by ignore_index...)?

Reply:
Yes as we discussed- I checked and IOU with ignored index behaviour is as right.
For binary segmentation though it is more common to just take the target IOU, but I'm also returning mIOU and BG IOU for logging.